static/styles/drafts.css

```diff
@@ -46,6 +46,53 @@
                 cursor: pointer;
             }
         }
+
+        .select-drafts-button {
+            position: absolute;
+            right: 35px;
+            top: 70px;
+            padding-right: 40px;
+            padding-top: 8px;
+
+            .fa-check-square {
+                position: absolute;
+                font-size: 18px;
+                top: 10px;
+                right: 7px;
+            }
+
+            .fa-square-o {
+                position: absolute;
+                font-size: 20px;
+                top: 9px;
+                right: 7px;
+            }
+
+            .button-text {
+                font-size: 14px;
+                padding-left: 8px;
+            }
+
+            .select-state-indicator {
+                padding-left: 5px;
+                padding-right: 8px;
+            }
+        }
+
```

Comment:
I think we are setting a lot of values position related values for `.delete-drafts-button` and `.select-drafts-button` in a hard coded manner by using `position: absolute`.  Which is probably the reason for position of select all drafts button to be unstable.

A more stable way to fix this could be to have drafts-header and drafts in two separate containers (divs) so that scrollbar only appears for drafts list and not for drafts header. Similar to what we do in other stream/organization settings modals.